infra/base-images/base-builder/compile

```diff
@@ -63,8 +63,21 @@ if [ $FUZZING_ENGINE = "none" ]; then
   export COVERAGE_FLAGS=
 fi
 
-export CFLAGS="$CFLAGS $SANITIZER_FLAGS $COVERAGE_FLAGS"
-export CXXFLAGS="$CFLAGS $CXXFLAGS_EXTRA"
+if [[ $FUZZING_LANGUAGE == "rust" ]]; then
+  # Rust does not support sanitizers and coverage flags via CFLAGS/CXXFLAGS.
+  # Instead, use RUSTFLAGS.
+  # FIXME: Support code coverage once support is in.
+  # See https://github.com/rust-lang/rust/issues/34701.
+  export RUSTFLAGS="--cfg fuzzing -Zsanitizer=${SANITIZER} -Cdebuginfo=1 -Cforce-frame-pointers"
```

Comment:
Hi !
Should we skip -Zsanitizer when $SANITIZER is "coverage" ?
It is not supported by rustc:
https://rustc-dev-guide.rust-lang.org/sanitizers.html

This build failure on cras happened recently: https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=22447#c2
Build log: https://oss-fuzz-build-logs.storage.googleapis.com/log-3e1434f6-4936-489f-ac03-690ebcbd0ebd.txt

```
Step #3: Caused by:
Step #3:   process didn't exit successfully: `rustc - --crate-name ___ --print=file-names --cfg fuzzing -Zsanitizer=coverage -Cdebuginfo=1 -Cforce-frame-pointers --crate-type bin --crate-type rlib --crate-type dylib --crate-type cdylib --crate-type staticlib --crate-type proc-macro --print=sysroot --print=cfg` (exit code: 1)
Step #3: --- stderr
Step #3: error: incorrect value `coverage` for debugging option `sanitizer` - one of: `address`, `leak`, `memory` or `thread` was expected
```
Could you please take a look?
Thanks!

Reply:
Sorry about that, excluded it in https://github.com/google/oss-fuzz/commit/d70af2fc967e103cd2e30b8a5df95697fffc5aec

Reply:
Thanks a lot for the help and quick fix!